camayoc/qcs_models.py

```diff
@@ -413,6 +414,9 @@ def __init__(
         # valid scan types are 'connect' and 'inspect'
         self.scan_type = scan_type
         self.options = {'max_concurrency': max_concurrency}
+        if disabled_optional_products:
```

Comment:
@kdelee I ran the scans test `test_scans_common.py` & with this updated version, they all passed.

Reply:
:+1: